modules/cluster-autoscaler-cr.adoc

```diff
@@ -74,7 +76,9 @@ If you do not specify a value, the default value of `1` is used.
 <17> Optional:  Specify the _node utilization level_. Nodes below this utilization level are eligible for deletion.
 +
 The node utilization level is the sum of the requested resources divided by the allocated resources for the node, and must be a value greater than `"0"` but less than `"1"`. If you do not specify a value, the cluster autoscaler uses a default value of `"0.5"`, which corresponds to 50% utilization. You must express this value as a string.
-<18> Optional: Specify any expanders that you want the cluster autoscaler to use.
+<18> In this section, you can specify the period to wait before scaling up nodes by using any valid link:https://golang.org/pkg/time/#ParseDuration[ParseDuration] interval, including `ns`, `us`, `ms`, `s`, `m`, and `h`. 
```

Comment:
i'm not sure how specific we want to get, but this value is how long the autoscaler should wait before reacting to a newly pending pod.